### ISSUE TITLE
I18n support?

### ISSUE THREAD

**Issue by ctjhoa** [offirgolan/ember-light-table#311]
Hi,
It seems that there is no [ember-i18n](https://github.com/jamesarosen/ember-i18n).
For example,  with `columns`:
```
import { translationMacro as t } from 'ember-i18n';
...
columns: computed(function() {
    return [{
      label: t('agreement.title'),
      valuePath: 'metadata.title'
    }];
  }),
```
`Assertion Failed: Cannot translate agreement.title. <(unknown mixin):ember591> does not have an i18n.`
Is this something on the roadmap or is there another way?

**Comment by ctjhoa** [comment]
I found a workaround but it's kinda ugly

```
titleHeader: t('agreement.title'),
columns: computed('titleHeader', function() {
    return [{
      label: get(this, 'titleHeader'),
      valuePath: 'metadata.title'
    }];
  }),
```

**Comment by offirgolan** [comment]
There is currently no out-of-the-box solution for this but it should be fairly easy to do by [creating your own column type](http://offirgolan.github.io/ember-light-table/docs/classes/Column.html#property_type) and overriding the label property. 

In your column definition, you can pass a `labelKey: 'agreement.title'` instead of `label: 'Foo Bar'` and then use that via `column.labelKey` in your column type.

**Comment by alexander-alvarez** [comment]
any plans to have 'official' `column types`, `cell-components`, etc that are used a lot by the community?
Or even just a poll of what people have implemented to improve documentation to show the different use cases that `ember-light-table` can achieve?

**Comment by ynnoj** [comment]
Import the `i18n` service in the component and you can use the `t` helper in the columns declaration.

```
{
  label: i18n.t('table.catalogue.name.key'),
  valuePath: 'name'
}
```

**Comment by offirgolan** [comment]
@alexander-alvarez Yeah I need to work on adding some more default cell & column types.
